models/public/googlenet-v4-tf/requirements.in

```diff
@@ -0,0 +1,2 @@
+tensorflow>=1.0.0,<2.0.0
+subprocess
```

Comment:
`subprocess` is a built-in module. (And if there's only one actual dependency, then using a requirements file is overkill. You could just say "run `pip install [...]`" in the documentation page.)